packages/components/src/inputs/src/TextArea.module.css

```diff
@@ -0,0 +1,57 @@
+.hop-TextArea  {
+    --hop-TextArea-gap: var(--hop-space-stack-xs);
+    --hop-TextArea-inline-size: 20rem;
+    --hop-TextArea-prefix-color: var(--hop-neutral-text-weak);
+    --hop-TextArea-input-padding: var(--hop-space-inset-xs) calc(var(--hop-space-inset-md) - var(--hop-space-inset-xs));
+    --hop-TextArea-char-count-padding: 0 calc(var(--hop-space-inset-md) - var(--hop-space-inset-xs)) var(--hop-space-inset-xs);
```

Comment:
Why do we need 12px here? We should be using 4 8 or 16px. Can we validate with design?

Reply:
because, this has nothing to do with design. The padding on the sides do end up adding up to the proper padding used in the design. I have just split it between the textarea and the input group and well the char count as well.
The reason for this is because I wanted the input group to have a little bit of padding for when the scrollbar is displayed (This will actually fix one of the issue we had with the scrollbar going outside of the corners).
I also didn't want all the padding to be on the outside of the html textarea because we need some padding inside due to the scrollbar.
Anyway, trust me, I think it looks much better like this.
oh another reason was for the little resize thing at the bottom right.